.babelrc

```diff
@@ -4,9 +4,10 @@
       "env",
       {
         "targets": {
-          "ie": 11,
           "browsers": [
-            "last 2 versions"
+            ">0.25%",
+            "not op_mini all",
+            "IE 11"
```

Comment:
http://browserl.ist/?q=%3E0.25%25 
ie 11 is already in

Reply:
Yes, but I added it the be future proof, when it falls under we have to ensure that we can remove the support before we really remove it